convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -207,13 +207,13 @@ def run(self):
                 description="Repository file packages which could not be removed",
                 diagnosis=message,
             )
-        else:
+        if pkgs_removed:
```

Comment:
Keep in mind that `pkgs_removed` wasn't declared outside of the try-catch, so this would fail if that ever changes. I would initialize pkgs_removed before it gets sorted

Reply:
But with the current state of the code wouldn't it be fine? 
This line `pkgs_removed = sorted(pkghandler.remove_pkgs_unless_from_redhat(pkgs_to_remove) or []) ` ensures that we either get the removed packages or an empty list.

Reply:
Okay that makes sense, let me initialize pkgs_removed